schema/feature_endpoint_mapping.json

```diff
@@ -225,6 +225,10 @@
       "provisioner"
     ]
   },
+  "/api/*/appliance_info": {
+    "feature_name": "appliance_info",
```

Comment:
Please add the new feature_name to https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/208830998/Unsupported+Features 
CC: @soniyamoholkar